crates/editor_core/src/gltf_unpack.rs

```diff
@@ -190,16 +187,23 @@ fn unpack_gltf(world: &mut World) {
 
 fn spawn_node(
     commands: &mut Commands,
-    node: &GltfNode,
+    node_handle: &Handle<GltfNode>,
     _gltf: &Gltf,
     ctx: &UnpackContext<'_>,
 ) -> Entity {
+    let gltf_nodes = ctx.gltf_nodes;
+    let node = match gltf_nodes.get(node_handle) {
```

Comment:
Replace this with
Let Some(node) = gltf_nodes.get(node_handle) else {
Log and return
}